libkbfs/tlf_handle.go

```diff
@@ -522,15 +522,13 @@ func (h *TlfHandle) GetCanonicalName() CanonicalTlfName {
 	return h.name
 }
 
-func buildCanonicalPath(public bool, canonicalName CanonicalTlfName) string {
-	var folderType string
+// buildCanonicalPath returns a canonical path for a tlf.
+func buildCanonicalPath(public bool, tlfName CanonicalTlfName) string {
```

Comment:
Do we need this method anymore?  Seems confusing to have it around with an identical but lower case name.  I think we can replace existing callers pretty easily.  Or maybe just rename it to something like `translatePublicAndBuildCanonicalPath()` and move it into the same file as `BuildCanonicalPath()`, if you don't feel like dealing with that.